packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.css

```diff
@@ -83,35 +83,18 @@
 .transaction-amount__token {
   display: flex;
   align-items: center;
-  color: var(--boxel-dark);
-
-  /* bordered display box */
   justify-content: center;
   padding: var(--boxel-sp-xl) var(--boxel-sp);
   border: 1px solid var(--boxel-light-400);
   border-radius: var(--boxel-border-radius);
 }
 
-.transaction-amount__token-icon {
-  align-self: flex-start;
-  flex-shrink: 0;
-  width: 2.5rem;
+.transaction-amount__token > div > :first-child {
   height: 2.5rem;
-  object-fit: contain;
-  margin-right: var(--boxel-sp-sm);
-}
-
-.transaction-amount__token-balance {
-  font-weight: 600;
-  font-size: 1.25rem;
-  line-height: calc(27 / 20);
-  letter-spacing: var(--boxel-lsp-xs);
-  margin-top: var(--boxel-sp-xxs);
+  display: flex;
+  align-items: center;
 }
 
-.transaction-amount__usd-balance {
-  color: var(--boxel-purple-500);
-  font: var(--boxel-font-sm);
-  letter-spacing: var(--boxel-lsp-xs);
-  margin-top: var(--boxel-sp-xs);
+.transaction-amount__token > div > :last-child {
+  text-align: center;
```

Comment:
Can we live without this styling? It being a `div > :last-child` seems a bit too generic to me.

with:
![image](https://user-images.githubusercontent.com/39579264/122725359-7c32af80-d2a7-11eb-8c37-1d39558ea464.png)


without:

![image](https://user-images.githubusercontent.com/39579264/122725242-5efde100-d2a7-11eb-8c25-6b1335a1e08e.png)

Reply:
The approximate sign on the USD is also removed, it might be a more important detail. Perhaps this component should also retain the original implementation instead?

Reply:
![image](https://user-images.githubusercontent.com/39579264/122725728-dd5a8300-d2a7-11eb-8128-55733dd71004.png)

Reply:
Left comments about the approximate sign on this. There actually isn't an approximate sign on this component and the original implementation's one was extraneous.